content/en/docs/intro/CheatSheet.md

```diff
@@ -0,0 +1,124 @@
+# Cheat Sheet - HELM
+
+Helm cheatsheet featuring  all the necessary commands required to  manage application through Helm.
```

Comment:
minor change: please change to "manage an application" and remove the extra whitespaces